Bottom: a2b662835892f889f5a56cc45c92bfa772dc171d
Top:    162b49ce4204f0b2bc37970e5b315d0fa6d8cd39
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 02:09:16 +0100

Refresh of win32-select-the-correct-gl-li

---

diff --git a/configure.ac b/configure.ac
index c5de7041..e9718a1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -427,7 +427,7 @@ Either do not use --enable-gl or enable the gtk HID.])
 	esac
 
 	GLU_CFLAGS=
-	GLU_LIBS=-lglu32 -lopengl32
+	GLU_LIBS="-lglu32 -lopengl32"
 
 	AC_DEFINE([ENABLE_GL], 1,
 		[Define to 1 if GL support is to be compiled in])
